src/app/screens/LNURLPay.tsx

```diff
@@ -47,7 +47,7 @@ function LNURLPay(props: Props) {
   const [successAction, setSuccessAction] = useState<
     LNURLPaymentSuccessAction | undefined
   >();
-
+  const [clear, setClear] = useState<boolean>(false);
```

Comment:
is this needed? 
if we make sure that `valueMSat` is never undefined but always a integer then we
can simple always do `setValueMSat(valueMSat + 10000);` I guess?

Reply:
The reason I added this was because currently `valueMSat` starts off with a value of the minimum sendable. So for me when I was testing there was already a value of 1 in the input. This made it so that when I clicked the buttons instead of going up in even numbers like 10 or 1000 it would do 11, 1001 etc. So I thought it would be cleaner to clear what is the value at the start so that when you use the buttons it reacts the way you would expect. Does this make sense?